controller/src/main/java/io/pravega/controller/util/Config.java

```diff
@@ -174,6 +174,18 @@ public final class Config {
     public static final Property<Integer> PROPERTY_RETENTION_BUCKET_COUNT = Property.named(
             "retention.bucket.count", 1, "retention.bucketCount");
 
+    public static final Property<String> PROPERTY_RETENTION_TYPE = Property.named(
+            "retention.type", "", "retention.type");
+
+    public static final Property<Integer> PROPERTY_RETENTION_MIN_VALUE = Property.named(
+            "retention.min.value", 0, "retention.minval");
+
+    public static final Property<Integer> PROPERTY_RETENTION_MAX_VALUE = Property.named(
+            "retention.max.value", 0, "retention.maxVal");
+
+    public static final Property<Boolean> PROPERTY_GLOBAL_RETENTION_POLICY = Property.named(
+            "retention.global.policy", false, "retention.bucketCount");
```

Comment:
I think this retention.bucketCount -> retention.globalPolicy

Reply:
nice catch, thanks :)